hamilton/lifecycle/default.py

```diff
@@ -512,13 +513,28 @@ def run_after_node_execution(
 SENTINEL_DEFAULT = None  # sentinel value -- lazy for now
 
 
+def accept_error_sentinels(func: Callable):
+    """Tag a function to allow passing in error sentinels.
+
+    For use with ``GracefulErrorAdapter``.
```

Comment:
Nit -- it's nice to add this to the docs -- here (https://github.com/DAGWorks-Inc/hamilton/blob/87c5de0a8a3ff8b31eb4946887735b46aa95fede/docs/reference/lifecycle-hooks/GracefulErrorAdapter.rst#L4). I'm happy to add it in afterwards cause it takes a bit of dev work to test out. Plus side is we can also cross-link the reference.

Reply:
If you don't mind adding that, I would appreciate the help. You'll get it done much faster, I think!

Reply:
You got it!